### DISCUSSION TITLE
Session id doesn't populates when creating logs from LWC.

### DISCUSSION THREAD

**Discussion by Suryansh-2301** [jongpie/NebulaLogger#677] [Q&A] [answered]
I have created a simple LWC component with a button and on clicking it button I want to create a log with a message and wants to capture session details as well. I m passing methodName as 'SYNCHROMOUS_DML'/'REST' in the saveLog method but doesn't seems to workout.

Note : I'm using the latest Nebula release i.e. 4.13.6 and it's unlocked.

HTML:
```
<template>
<c-logger></c-logger>
<h1>Nebula Logger Test</h1>
<lightning-button variant="base" label="labelName" title="titleName" onclick={handleClick}></lightning-button>
</template>
```

JS:
```
import { LightningElement } from 'lwc';
export default class TestNebula extends LightningElement {
    handleClick(){
        const logger = this.template.querySelector('c-logger');

        logger.info('Button Clicked');
        logger.saveLog('SYNCHRONOUS_DML');
    }
}
```
js-meta.XML

```
<?xml version="1.0"?>
<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
	<apiVersion>57.0</apiVersion>
	<isExposed>true</isExposed>
	<targets>
		<target>lightning__RecordPage</target>
		<target>lightning__AppPage</target>
		<target>lightning__HomePage</target>
	</targets>
</LightningComponentBundle>
```

**Comment by jongpie** [comment]
@Suryansh-2301 thanks for reporting this! I did some quick testing, and was able to reproduce the issue. I think this is a bug - I've opened issue #678 to track it, hopefully this will be a fairly quick item to fix.

**Reply by jongpie** [reply, marked as answer]
@Suryansh-2301 this is now fixed in [release `v4.13.10`](https://github.com/jongpie/NebulaLogger/releases/tag/v4.13.10) - you should now (hopefully 😅) start to see the session ID/session data populated when creating logs from LWC.

Let me know if you run into any other issues!

**Reply by Suryansh-2301** [reply]
Thanks!! @jongpie for the fix I have verified and now it works fine.

**Reply by jongpie** [reply]
@Suryansh-2301 that's great news, thanks for verifying!
